test/prob/gamma/gamma_ccdf_log_test.hpp

```diff
@@ -32,11 +32,11 @@ class AgradCcdfLogGamma : public AgradCcdfLogTest {
     ccdf_log.push_back(
         std::log(1.0 - 0.6321205588285576659757));  // expected ccdf_log
 
-    param[0] = 16.0;  // y
-    param[1] = 3.0;   // alpha
-    param[2] = 3.0;   // beta
+    param[0] = 9.0;  // y
```

Comment:
I made the test case easier because the test implementations were struggling numerically. See: https://github.com/stan-dev/math/issues/2006

The Stan implementations of the ccdf didn't seem to have this problem.